homeassistant/components/http.py

```diff
@@ -313,6 +371,11 @@ def write_json(self, data=None, status_code=HTTP_OK, location=None):
         if location:
             self.send_header('Location', location)
 
+        if self._session is not None:
+            self.send_header(
+                'Set-Cookie',
+                'sessionId='+self.cookie['sessionId'].value)
```

Comment:
It seems that the only use for the cookie object is to pass the sessionId between `get_session` and `set_session` and this method. Why not get it from the session object?

Also, it seems like we're sending a set cookie header every time.